inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -303,7 +303,7 @@ def get_query_related_info(option: str, info: Dict) -> Dict:
                 del info['values'][elem]
     elif option == "version":
         info = {}
-        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()        
+        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
     elif option == "all":
```

Comment:
**<h2>$\color{#FFD700}Coding Style: CodeConsistency$</h2>**
**Suggestion:** Ensure consistent use of single quotes for strings throughout the codebase to maintain code style consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                logger.debug('Querying diagnostic for container health.')
                ...
                logger.info('Message received: %s on topic: %s', payload, topic)
                ...
                logger.debug(
                    'Received an unexpected second response to container health command. Ignoring.')
```

Reply:
good suggestion--we probably want to reformat with 'black' which enforces this